install.rdf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The element listed is a required element in the install manifest specification. It must be added to your addon. Missing elements: targetApplication